terraform/shared/shared.tf.json.template.py

```diff
@@ -12,7 +12,18 @@
     block_public_s3_bucket_access,
 )
 
+
 emit_tf(block_public_s3_bucket_access({
+    **({
+           'data': {
+               'local_file': {
```

Comment:
This introduces usage of the `local` provider. It's version must be pinned in the template for `providers.tf.json`. Then update the lock file using the workaround mentioned in https://github.com/DataBiosphere/azul/issues/4751 and commit the result. Note that the lock file is symlinked from `terraform/shared` to `terraform`, and should remain so.

However, this might not be needed after all. See below.